modules/tide_ui_restriction/tide_ui_restriction.permissions.yml

```diff
@@ -4,3 +4,12 @@ access admin_config pages:
 'access event log track':
   title: 'Access event log track'
   description: 'Allow user to access event log track.'
+access user creation page:
```

Comment:
Was this the bit that was needed to allow user creation? i.e. the permission was fine but we were restricting the route?